sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -424,10 +505,10 @@ await using (var scope = await EventHubScope.CreateAsync(1))
                 var eventBatch = new[]
                 {
                     // Actual limit is 1046520 for a single event
-                    new EventData(new byte[1000000])
+                    new EventData(new byte[100000])
```

Comment:
Why are we changing this value? 10^6 should work and it's not too close to the limit.

Reply:
In short:  nightly failures causing timeouts on non-Windows OS at a 5 min timeout period.  I'm willing to sacrifice some pushing on the boundaries to avoid extending the nightly runs.   It's the job of the service and underlying AMQP link negotiation to determine and enforce the size limit, not a client responsibility.